### PR TITLE
Issue 1008

### PR DESCRIPTION
**Pull request by mphan6**
## Details
**Issue you fixed:** #1008 

**Issue you added:** #1044 

**Name of the function you implemented**: nthSumNine

**Name of the function you added test cases for**: firstDigitSum

## Checklist (won't be merged if they are not all ticked)
- [x] I have asked for this issue, and was assigned it
- [x] I have properly filled the "Details" section above  :arrow_up:
- [x] I didn't modify any test case  :red_circle:
- [x] I have implemented the function and all its tests pass  :white_check_mark:
- [x] I have added a new test case for someone to implement  :new:
- [x] I have created an issue for my new test case  :speak_no_evil:
- [x] I've :star: the repository (or not, all up to you)
- [x] And more importantly, I've had fun! :beer:

### CODE REVIEW

**Comment by mphan6** [comment]
As per the discussion within the issue #1008 - the test cases were incorrect and I was given a new set to use.

**Comment by rgehan** [comment]
Hey @jeanbenitez,

We cannot let you merge your PR as it doesn't pass the tests.

If you think this is a mistake, feel free to ping me in your reply.

##### This comment was automatically generated by a bot.

**Comment by jeanbenitez** [comment]
@rgehan please check this case

File: src/firstDigitSum.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  it('firstDigitSum does this thing...', () => {
+    expect(firstDigitSum(100008)).toEqual(9);
+    expect(firstDigitSum(325162)).toEqual(19);
+    expect(firstDigitSum(99999)).toEqual(45);
+    expect(firstDigitSum(10636408)).toEqual(28);
+  });
+
+  it('firstDigitSum invalid args', () => {
+    expect(() => firstDigitSum()).toThrow();
+    expect(() => firstDigitSum(0)).toThrow();
```
Comment (jeanbenitez) [inline src/firstDigitSum.test.js]: I think you need delete this cases, this don't apply in your kata

File: src/firstDigitSum.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    expect(() => firstDigitSum(51)).toThrow();
```
Comment (jeanbenitez) [inline src/firstDigitSum.test.js]: I think you need delete this cases, this don't apply in your kata

Comment (jeanbenitez) [reply src/firstDigitSum.test.js]: the 51 and 0 values

Comment (mphan6) [reply src/firstDigitSum.test.js]: I did that, in the other, and messed up this time around haha..

**Comment by mphan6** [comment]
I believe everything is fixed now! It's just the tester file that was modified due to incorrect test cases.

**Comment by jeanbenitez** [comment]
@mphan6 Try to make the comment I made here https://github.com/rgehan/hacktoberfest-2k18-katas/pull/1045#issuecomment-432854752

**Comment by mphan6** [comment]
@jeanbenitez I did, didn't i? I removed the bad test cases.. o=

**Comment by rgehan** [comment]
Hey @mphan6,

We cannot let you merge your PR as it doesn't pass the tests.

If you think this is a mistake, feel free to ping me in your reply.

##### This comment was automatically generated by a bot.

**Comment by mphan6** [comment]
@rgehan Hello, mind checking the PR?

**Comment by rgehan** [comment]
Hey @mphan6,

Thanks for your contribution, your PR was merged, and your kata is now available for others to play with.

If there's been any issue with the process, please ping me in your reply.

##### This comment was automatically generated by a bot.

**Pull request merged**